app/models/S4LModels.scala

```diff
@@ -83,3 +84,9 @@ case class S4LVatEligibility
 (
   vatEligibility: Option[VatServiceEligibility]
 )
+
+case class S4LVatLodgingOfficer
+(
+  officerHomeAddressView: Option[OfficerHomeAddressView]
```

Comment:
i assume we are adding more elements to the case class soon. we don't want single-element product types